torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -1757,6 +1757,7 @@
   name: eye
   backends:
     - CPU
+    - CUDA
```

Comment:
Are we going to make `THD` follow a similar pattern as ATen, where we don't need to manually specify the wrapper functions?